aspnetcore/performance/response-compression.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Response Compression Middleware | Microsoft Docs
+author: guardrex
+description: An introduction to response compression with instructions on how to use Response Compression Middleware in ASP.NET Core applications.
+keywords: ASP.NET Core, performance, response compression, gzip, accept-encoding, middleware
+ms.author: riande
+manager: wpickett
+ms.date: 01/12/2017
+ms.topic: article
+ms.assetid: de621887-c5c9-4ac8-9efd-f5cc0457a134
+ms.technology: aspnet
+ms.prod: aspnet-core
+uid: performance/response-compression
+---
+# Response Compression Middleware
+
+By [Luke Latham](https://github.com/GuardRex)
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/performance/response-compression/sample)
+
+This document introduces response compression and explains how to use ASP.NET Core Response Compression Middleware. You can explore the features of the Response Compression Middleware with the [sample application](https://github.com/aspnet/Docs/tree/master/aspnetcore/performance/response-compression/sample). The sample illustrates the compression of application responses using Gzip and custom compression providers. It also shows you how to add a MIME type to the default list of MIME types for compression.
```

Comment:
on 2nd pass - why should I read this article? I'd make my first sentence:

Response compression can reduce the payload sent over the wire and increase the responsiveness of your app, sometimes dramatically.

Then I'd cut most of that from the next paragraph. Now folks have a reason to read this article (until they get the **new**) next section that tells most of them they don't need it :--)